salt/states/rsync.py

```diff
@@ -97,7 +99,10 @@ def synchronized(name, source,
     '''
     ret = {'name': name, 'changes': {}, 'result': True, 'comment': ''}
 
-    if not os.path.exists(source):
+    if name.strip() == '/':
```

Comment:
HI @cachedout 

 Doing an rsync against /  will probably lead to a broken system. The idea here was to avoid some typo or missing that in a pillar, for example and running a rsync with --delete against / and broke a system.
 I can remove that or add something like

```python

if name.strip() == '/' and force and not delete

```

Anyway I can revert this and left the other part of the PR to make this working. Now is not working at all.

Thanks.